graphql2/graphqlapp/integrationkey.go

```diff
@@ -58,3 +59,51 @@ func (key *IntegrationKey) Href(ctx context.Context, raw *integrationkey.Integra
 
 	return "", nil
 }
+
+func (q *Query) IntegrationKeys(ctx context.Context, input *graphql2.IntegrationKeySearchOptions) (conn *graphql2.IntegrationKeyConnection, err error) {
+	if input == nil {
+		input = &graphql2.IntegrationKeySearchOptions{}
+	}
+
+	var opts integrationkey.InKeySearchOptions
+	if input.Search != nil {
+		opts.Search = *input.Search
+	}
```

Comment:
Currently, when there's no results for the search input, it returns a graphql error: 
<img width="353" alt="graphql error" src="https://user-images.githubusercontent.com/52386267/187958036-40be1ff6-9f97-4dbc-a851-06918ba6af29.png">

Suggested code change to return "No options" instead:
<img width="365" alt="Screen Shot 2022-09-01 at 10 51 34 AM" src="https://user-images.githubusercontent.com/52386267/187958179-60db8a1a-b8a3-472a-86a1-721b12e47bf3.png">



```suggestion
	}
	conn = new(graphql2.IntegrationKeyConnection)
	err = validate.UUID("Search", opts.Search)
	if err != nil {
		conn.Nodes = []integrationkey.IntegrationKey{}
		return conn, nil
	}
 
```

Reply:
Added pattern matching in the search query that will match the key id while typing and return 'No options' if there are no matches.

Reply:
:brain: